package.json

```diff
@@ -15,26 +15,26 @@
     "eslint": "^4.19.1",
     "eslint-config-prettier": "^2.8.0",
     "eslint-config-standard": "^11.0.0-beta.0",
-    "eslint-plugin-import": "^2.12.0",
-    "eslint-plugin-mocha": "^5.0.0",
+    "eslint-plugin-import": "^2.13.0",
+    "eslint-plugin-mocha": "^5.1.0",
     "eslint-plugin-node": "^6.0.1",
-    "eslint-plugin-prettier": "^2.6.1",
+    "eslint-plugin-prettier": "^2.6.2",
     "eslint-plugin-promise": "^3.8.0",
     "eslint-plugin-standard": "^3.1.0",
     "faker": "^4.1.0",
     "husky": "^0.14.3",
     "lint-staged": "^7.2.0",
-    "prettier": "^1.13.5",
+    "prettier": "^1.13.7",
     "solhint": "^1.2.1",
-    "solidity-coverage": "^0.5.4"
+    "solidity-coverage": "^0.5.5"
   },
   "scripts": {
     "precommit": "lint-staged",
     "prettier": "prettier --single-quote --no-semi --write '**/*.{js,json}'",
     "eslint-check": "eslint --print-config .eslintrc.js | eslint-config-prettier-check",
     "test": "truffle test",
     "test:cov": "solidity-coverage",
-    "lint:sol": "solhint contracts/**/*.sol",
+    "lint:sol": "solhint contracts/**/**/*.sol",
```

Comment:
npm run lint:sol wasn't working with all contracts until I added this. Not sure what is up.